core/src/main/java/discord4j/core/state/StateHolder.java

```diff
@@ -63,6 +63,7 @@
     private final Store<LongLongTuple2, PresenceData> presenceStore;
     private final LongObjStore<RoleData> roleStore;
     private final LongObjStore<UserData> userStore;
+    private final Store<String, TemplateData> templateStore;
```

Comment:
A cache is not needed for entities we don't get gateway updates for. Looks like this is the case unless I'm missing something, for instance, is it going to be useful for 3.2 migration?

Reply:
This item is still unresolved since we will not cache these entities. Please treat them like webhooks (requests directly to REST API)